upstream-community-operators/seldon-operator/seldonoperator.crd.yaml

```diff
@@ -12,5 +12,5 @@ spec:
     - sdep
   scope: Namespaced
   version: 0.1.2
```

Comment:
spec.versions[0].name: Invalid value: "0.1.2"
spec:version: Each API Group can exist in multiple versions. Use the version name in the URL, for example v1alpha1, v1beta1, or v1. For more details, see the Consideration section. For more information on versioning CRDs, see the Versioning section.
Read here: https://github.com/kyma-project/community/blob/master/guidelines/custom-resource-definitions.md#location-and-file-name